controller/db/terminator_store_test.go

```diff
@@ -92,6 +119,8 @@ func (ctx *TestContext) createTestTerminators() *terminatorTestEntities {
 	e.terminator2.Id = uuid.New().String()
 	e.terminator2.Service = e.service.Id
 	e.terminator2.Router = e.router2.Id
+	e.terminator2.Binding = uuid.New().String()
+	e.terminator2.Address = uuid.New().String()
```

Comment:
Weren't we using shortids for these? It's just a `_test`, but...

Reply:
These aren't ids, I just need some random text and uuid is a fast way to get some. Even in the cases where it is used for non-terminator ids, it's used for generating invalid ids (as opposed to unset ids). I removed the use for terminator ids